package.json

```diff
@@ -91,7 +91,7 @@
       "prettier --write"
     ]
   },
-  "//": "tar is not a direct dependency of our project; we’ve only added it here to remove a security vulnerability introduced by a transitive dependency. This should be removed; see IC-2177",
+  "//": "validator is pinned for CVE-2021-3765; remove when fixed upstream",
```

Comment:
Is it worth making this a `TODO` so that it shows up when searching for things we need to look at?

Reply:
👍 added in c2562d1500f2bf1fdd99819c1c24925038e7b39c